x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeService.java

```diff
@@ -247,6 +251,26 @@ public void execute(
         }
     }
 
+    private static void updateExecutionInfoAfterCoordinatorOnlyQuery(long queryStartNanos, EsqlExecutionInfo execInfo) {
+        long tookTimeNanos = System.nanoTime() - queryStartNanos;
+        execInfo.overallTook(new TimeValue(tookTimeNanos, TimeUnit.NANOSECONDS));
+        if (execInfo.isCrossClusterSearch()) {
+            for (String clusterAlias : execInfo.clusterAliases()) {
+                // The local cluster 'took' time gets updated as part of the acquireCompute(local) call in the coordinator, so
+                // here we only need to update status for remote clusters since there are no remote ComputeListeners in this case.
```

Comment:
It's probably worth saying something about how we didn't need the remote clusters at all in this situation.

I'm not sure how you can write an ESQL that does this by the way.

Reply:
Ah! That's right. `LIMIT 0`. We resolve the remote clusters so we can get type information from them, but then we never need them.

Reply:
I'll update the code comment to specifically mention the "LIMIT 0" usage so it's clear to future readers/maintainers.